backend/de.metas.manufacturing.rest-api/src/main/java/de/metas/manufacturing/job/service/commands/ReceiveGoodsCommand.java

```diff
@@ -252,6 +256,14 @@ private void assertMixingDifferentProductsAllowed(@NotNull final I_M_HU targetHU
 		}
 	}
 
+	private void assertIfDestroyed(@NotNull final I_M_HU targetHU)
```

Comment:
what exactly means "assert if destroyed"?

pls rename it to "assertNotDestroyed"

but if i am looking on other example, e.g. de.metas.handlingunits.picking.candidate.commands.AddQtyToHUCommand#createAllocationDestination

there we assert status to be active.
why not here too?